src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/custom.py

```diff
@@ -1279,7 +1280,7 @@ def create_vmss(vmss_name, resource_group_name, image,
                 location=None, tags=None, upgrade_policy_mode='manual', validate=False,
                 admin_username=getpass.getuser(), admin_password=None, authentication_type=None,
                 vm_sku="Standard_D1_v2", no_wait=False,
-                ssh_dest_key_path=None, ssh_key_value=None,
+                ssh_dest_key_path=None, ssh_key_value=None, generate_ssh_keys=False,
```

Comment:
Recommend changing the flag name to generate_ssh_key (singular) since at most only one key would be generated.